apps/nextjs/.storybook/decorators/LessonPlanTrackingDecorator.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import type { Decorator } from "@storybook/react";
+
+import { lessonPlanTrackingContext } from "../../src/lib/analytics/lessonPlanTrackingContext";
+
+export const LessonPlanTrackingDecorator: Decorator = (Story) => (
+  <lessonPlanTrackingContext.Provider
```

Comment:
Seems a little strange to have this lowercase

Reply:
The react docs use a capital for this, so I'll update